packages/react/src/components/Tabs/Tabs.tsx

```diff
@@ -85,17 +74,28 @@ const Tabs = ({
   };
 
   const tabComponents = tabs.map((child, index) => {
-    const { ...other } = (child as React.ReactElement<any>).props;
+    const { targetref, id: propId, ...other } = (child as React.ReactElement<
+      any
+    >).props;
     const selected = index === activeIndex;
+    const [id] = propId ? [propId] : useId(1, 'tab');
+
+    useEffect(() => {
+      targetref.current?.setAttribute('aria-controlledby', id);
```

Comment:
This isn't a valid aria property

Reply:
I am setting aria-controls here.
<img width="443" alt="Screen Shot 2021-10-07 at 3 08 09 PM" src="https://user-images.githubusercontent.com/83524297/136447175-923f280a-4967-42f1-815a-36b0e0ff4c06.png">
›